lib/plugins/aws/deploy/lib/iam-role-lambda-execution-template.json

```diff
@@ -16,6 +16,31 @@
           ]
         }
       ]
-    }
+    },
+    "Policies": [
+      {
+        "PolicyName": "[TO BE REPLACED]",
+        "PolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Action": [
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream"
+              ],
+              "Resource": []
+            },
+            {
+              "Effect": "Allow",
+              "Action": [
+                "logs:PutLogEvents"
+              ],
+              "Resource": []
+            }
+          ]
+        }
+      }
+    ]
```

Comment:
@eahefnawy using two was important because the use of star would give creation rights to any log group with the same prefix.  but as you note, it is legacy that can be removed now that explicit log group creation happens.

There's a bit more paranoia not in there though...  Here is our latest:
```
                - Effect: Allow
                  Action:
                    - logs:CreateLogStream
                  Resource:
                    - Fn::GetAtt:
                      - TheLogGroup
                      - Arn
                - Effect: Allow
                  Action:
                    - logs:PutLogEvents
                  Resource:
                    - Fn::Join:
                      - ":"
                      - - Fn::GetAtt:
                          - TheLogGroup
                          - Arn
                        - "*"
```
This way, the lambda can't create log streams in any similarly-prefixed-but-not-the-log-group-that-has-been-created while still being able to create in the created log group and then write to any such log stream.